src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/Models/ReindexJobRecord.cs

```diff
@@ -80,11 +80,13 @@ protected ReindexJobRecord()
         [JsonProperty(JobRecordProperties.FailureCount)]
         public ushort FailureCount { get; set; }
 
+#pragma warning disable CA1002
```

Comment:
Can we add a comment for why the rule is disabled here?

Reply:
Or I can be less lazy and actually convert it into collection...